README.md

```diff
@@ -380,14 +380,30 @@ Useful in [Continuous integration (CI)](https://en.wikipedia.org/wiki/Continuous
 #### --cd-version
 
 ```sh
-$ lerna publish --cd-version (patch | major | minor)
+$ lerna publish --cd-version (patch | major | minor | prepatch | premajor | preminor | prerelease)
```

Comment:
I prefer the ordering in the [other PR](https://github.com/lerna/lerna/pull/958/files#diff-04c6e90faac2675aa89e2176d2eec7d8), patch -> minor -> major